src/Peachpie.Library.MySql/MySqlConnectionResource.cs

```diff
@@ -48,7 +48,7 @@ public override void ClosePendingReader()
             var myreader = (MySqlDataReader)_pendingReader;
             if (myreader != null)
             {
-                myreader.Close();   // we have to call Close() on MySqlDataReader, it is declared as non-virtual!
```

Comment:
Is this safe now? In recent version of MySql connector, `Dispose()` didn't close the underlaying connection causing huge resources leaking and eventually applications to stop being able to open new connections.

Reply:
The underlying connection will be closed if we set CommandBehavior.CloseConnection flag when creating the command. Otherwise, the pooling behavior specified in the connection string should control the lifetime of the connections.

https://github.com/mysql-net/MySqlConnector/blob/master/src/MySqlConnector/MySql.Data.MySqlClient/MySqlDataReader.cs#L285

Reply:
good it seams they fixed this